### ISSUE TITLE
Reopening .localstorage file may lead to JsonReaderException

### ISSUE THREAD

**Issue by hanssens** [hanssens/localstorage#19]
Reopening an existing .localstorage file may lead to the following exception:

> JsonReaderException : Additional text encountered after finished reading JSON content

Stacktrace:

```
Newtonsoft.Json.JsonReaderException : Additional text encountered after finished reading JSON content: t. Path '', line 1, position 67.
   at Newtonsoft.Json.JsonTextReader.Read()
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.Deserialize(JsonReader reader, Type objectType, Boolean checkAdditionalContent)
   at Newtonsoft.Json.JsonSerializer.DeserializeInternal(JsonReader reader, Type objectType)
   at Newtonsoft.Json.JsonConvert.DeserializeObject(String value, Type type, JsonSerializerSettings settings)
   at Newtonsoft.Json.JsonConvert.DeserializeObject[T](String value, JsonSerializerSettings settings)
```

**Comment by hanssens** [comment]
Still unknown when this occurs, but the cause being that the JSON structure might become corrupted. This is an example output:

```
{
	"foo":"175690",
	"bar":"\"U20521\"",
	"from":"\"2018-05-30T10:37:45.2329671+02:00\"",
	"until":"\"2018-06-29T10:37:45.3339772+02:00\""
}
}
```

... which actually should be:

```
{
	"foo":"175690",
	"bar":"\"U20521\"",
	"from":"\"2018-05-30T10:37:45.2329671+02:00\"",
	"until":"\"2018-06-29T10:37:45.3339772+02:00\""
}
```

/cc Thanks @bryandh, for reporting this as well.

**Comment by hanssens** [comment]
May have been fixed in the latest v2.0.0-beta-2 release, which is [now published on NuGet](https://www.nuget.org/packages/LocalStorage/2.0.0-beta-2).

**Comment by hanssens** [comment]
Confirmed. This fixes it.
